mypy.ini

```diff
@@ -2,4 +2,19 @@
 ignore_missing_imports = True
 
 [mypy-tensorflow.*]
+ignore_missing_imports = True
+
+[mypy-srsly.*]
+ignore_missing_imports = True
+
+[mypy-flair.*]
+ignore_missing_imports = True
+
+[mypy-sklearn.*]
+ignore_missing_imports = True
+
+[mypy-tqdm.*]
+ignore_missing_imports = True
+
+[mypy-torch.*]
```

Comment:
have you checked that each of those packages do not have mypy bindings in some oss packages?

Reply:
Torch has own mypy binding.  Therefore should it be removed from ignore_missing_imprors ?